src/pkg/registry/client.go

```diff
@@ -53,6 +54,13 @@ var (
 		schema2.MediaTypeManifest,
 		schema1.MediaTypeSignedManifest,
 	}
+
+	localRegistryURL = map[string]bool{
```

Comment:
I don't think we can use hard-code here.

Reply:
oh, you mean hardcode here. We can initialize this map from config.